### ISSUE TITLE
Can you override an app at Site Collection level

### ISSUE THREAD

**Issue by simesy** [SharePoint/sp-dev-docs#7925]
Globally my tenant is using PnP Search v4.4.0. I want to run v4.6.1 (latest). When i add the app I don't see the dialog to deploy it and I don't see any changes reflected when I use the web parts.

Can you clarify how App Catalog works (or doesn't work) in this case?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fb134fd8-6880-360a-faea-7eb0eedca2d9
* Version Independent ID: 16f799b7-deff-b125-ceed-343933bcbe79
* Content: [Use the site collection app catalog](https://docs.microsoft.com/en-us/sharepoint/dev/general-development/site-collection-app-catalog)
* Content Source: [docs/general-development/site-collection-app-catalog.md](https://github.com/SharePoint/sp-dev-docs/blob/master/docs/general-development/site-collection-app-catalog.md)
* Product: **sharepoint**
* Technology: **sharepoint-framework**
* GitHub Login: @VesaJuvonen
* Microsoft Alias: **vesaj**

**Comment by bcameron1231** [comment]
Yes you can, by enabling and using the Site Collection App Catalog and deploying the .sppkg file for the v4.6.1 version.

Here is a link to the documentation
https://docs.microsoft.com/en-us/sharepoint/dev/general-development/site-collection-app-catalog
